app/views/layouts/exception_hunter/application.html.erb

```diff
@@ -24,7 +24,7 @@
       <div class="row">
         <div class="column column-40">
           <%= link_to errors_path do %>
-            <div class="nav__title">Exception Hunter</div>
+            <div class="nav__title"><%= Rails.application.class.module_parent_name %></div>
```

Comment:
I think something like 
```suggestion
            <div class="nav__title">Exception Hunter / <%= Rails.application.class.module_parent_name %></div>
```

would look better

Reply:
Will update it and attach a screen shot